app/support/notification_sender.rb

```diff
@@ -7,8 +7,17 @@ def initialize(current_facility, order_detail_ids)
     @order_detail_ids = order_detail_ids
   end
 
+  def init_account_ids_to_notify
+    @account_ids_to_notify =
```

Comment:
Would this be cleaned up at all if we turned `account_ids_to_notify` into a memoized method rather than an `attr_reader` with an initialization?